### ISSUE TITLE
Can we use QR code for all tasks as Bypassing ASR?

### ISSUE THREAD

**Issue by kinmaru** [RoboCupAtHome/RuleBook#668]
The rulebook says the following description about QR code.
・3.4.3 "However, artificial markers (e.g. bar codes, QR codes, OpenCV markers) are strictly forbidden."
・3.7.8 "Ad-hoc start signal: Other means of triggering robot to action are allowed but must be approved by the Technical Committee (TC) during the Robot Inspection (see Section 4.4). These include: QR Codes"
・3.8.3 "QR Codes: A reduction of 30% of the maximum attainable score is applied when a QR code is used to instruct the robot."

I saw some tasks don't refer to Bypassing ASR in the rulebook, but can we use QRcode as Bypassing ASR for all tasks?

If we could use QR code for all tasks as Bypassing ASR for all tasks, can we let the operator use the QRcode application we made(of course, it's easy to use for everyone)?

**Comment by kyordhel** [comment]
# TL;DR: Yes. 

Sections 3.4.3 and 3.7.8 are irrelevant to your question. 

What you propose is feasible but highly impractical, hence unadvised. If you plan to rely on an app, rather skip QR codes and use a more efficient and intuitive way of interaction. QR codes should be a last resource for unskippable steps such as GPSR command retrieval. Typing in is time consuming.
